src/main/java/homework/Main.java

```diff
@@ -0,0 +1,11 @@
+package homework;
+
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+public class Main {
+    public static void main(String[] args) {
+        AnnotationConfigApplicationContext context = new AnnotationConfigApplicationContext(JavaConfig.class);
+        Time bean = context.getBean(Time.class);
```

Comment:
Bean is a very common name for var. Try to find the name that describes this var.